lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -975,3 +975,127 @@ def assert_page_correct(page, page_size, expected_start, expected_stop, expected
         # Page past the end
         with self.assertRaises(Http404):
             self.get_comment_list(thread, endorsed=True, page=2, page_size=10)
+
+
+class CreateThreadTest(CommentsServiceMockMixin, UrlResetMixin, ModuleStoreTestCase):
+    """Tests for create_thread"""
+    @mock.patch.dict("django.conf.settings.FEATURES", {"ENABLE_DISCUSSION_SERVICE": True})
+    def setUp(self):
+        super(CreateThreadTest, self).setUp()
+        httpretty.reset()
+        httpretty.enable()
+        self.addCleanup(httpretty.disable)
+        self.maxDiff = None  # pylint: disable=invalid-name
```

Comment:
optional: make a note that this is irrelevant to the test logic, and only affects output in the case of failures.  (also, could it be defined as a class attribute instead of in setUp?)